test/testSyntax.js

```diff
@@ -14,12 +14,28 @@ require('source-map-support').install()
 //
 //
 
-const tap = require('tap-lite-tester')
+import tap from 'tap-lite-tester'
```

Comment:
When I ran the tests, I had to use the `require` include instead of the new `import` syntax. Apparently some [dependency of babel-preset-jsy](https://github.com/shanewholloway/babel-preset-jsy/blob/master/package.json#L27-L28) adds this capability to the full stack, but it isn't present in this lower-level plugin.